src/context/errorContext/errorContextProvider.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  FC,
+  ReactNode,
+  memo,
+  useMemo,
+  useState,
+} from 'react';
+import { ErrorContext, ErrorContextProps } from './errorContext';
+
+interface Props {
+  children: ReactNode,
+}
+
+export const ErrorContextProvider: FC<Props> = memo(({ children }) => {
+  const [errorMessage, setErrorMessage] = useState('');
```

Comment:
The default value of errorMessage is an empty string. It might be more clear to make the default value `null` to indicate that there is no error.